specs/experimental/standard-l2-genesis.md

```diff
@@ -214,22 +240,27 @@ specific configuration out of the initial L2 genesis state. All network specific
+| ----------------------- | ------------------------ | ------------------------------------------------------------------------------------------------------------------ |
+| `ConfigType`            | `uint8`                  | An enum representing the type of config being set                                                                  |
+| `WithdrawalNetwork`     | `uint8(0)` or `uint8(1)` | `0` means withdraw to L1, `1` means withdraw to L2                                                                 |
+| `RECIPIENT`             | `address`                | The account that will receive funds sent out of the `FeeVault`                                                     |
+| `MIN_WITHDRAWAL_AMOUNT` | `uint256`                | The minimum amount of native asset held in the `FeeVault` before withdrawal is authorized                          |
+| Fee Vault Config        | `bytes32`                | `bytes32((WithdrawalNetwork << 248) \|\| uint256(uint88(MIN_WITHDRAWAL_AMOUNT)) \|\| uint256(uint160(RECIPIENT)))` |
 
-| Name | Value | Definition |
-| --------- | ------------------------- | -- |
-| `ConfigType` | `uint8` | An enum representing the type of config being set |
-| `WithdrawalNetwork` | `uint8(0)` or `uint8(1)` | `0` means withdraw to L1, `1` means withdraw to L2  |
-| `RECIPIENT` | `address` | The account that will receive funds sent out of the `FeeVault` |
-| `MIN_WITHDRAWAL_AMOUNT` | `uint256` | The minimum amount of native asset held in the `FeeVault` before withdrawal is authorized |
-| Fee Vault Config | `bytes32` | `bytes32((WithdrawalNetwork << 248) \|\| uint256(uint88(MIN_WITHDRAWAL_AMOUNT)) \|\| uint256(uint160(RECIPIENT)))` |
-| `BASE_FEE_VAULT_CONFIG` | `bytes32(uint256(keccak256("opstack.basefeevaultconfig")) - 1)` | The Fee Vault Config for the `BaseFeeVault` |
-| `L1_FEE_VAULT_CONFIG` | `bytes32(uint256(keccak256("opstack.l1feevaultconfig")) - 1)` | The Fee Vault Config for the `L1FeeVault` |
-| `SEQUENCER_FEE_VAULT_CONFIG` | `bytes32(uint256(keccak256("opstack.sequencerfeevaultconfig")) - 1)` | The Fee Vault Config for the `SequencerFeeVault` |
+### L1Block slots
+
+| Name                                | Value                                                                      | Definition                                         |
+| ----------------------------------- | -------------------------------------------------------------------------- | -------------------------------------------------- |
+| `BASE_FEE_VAULT_CONFIG`             | `bytes32(uint256(keccak256("opstack.basefeevaultconfig")) - 1)`            | The Fee Vault Config for the `BaseFeeVault`        |
+| `L1_FEE_VAULT_CONFIG`               | `bytes32(uint256(keccak256("opstack.l1feevaultconfig")) - 1)`              | The Fee Vault Config for the `L1FeeVault`          |
+| `SEQUENCER_FEE_VAULT_CONFIG`        | `bytes32(uint256(keccak256("opstack.sequencerfeevaultconfig")) - 1)`       | The Fee Vault Config for the `SequencerFeeVault`   |
```

Comment:
When we land the operator fee in https://github.com/ethereum-optimism/optimism/pull/12166, we will need to rebase on top of that and then add that to the specs, both here and in the mermaid diagram